build.sbt

```diff
@@ -219,7 +210,6 @@ val `polynote-server` = project.settings(
 ).dependsOn(`polynote-runtime` % "provided", `polynote-runtime` % "test", `polynote-kernel` % "provided", `polynote-kernel` % "test->test")
 
 val sparkVersions = Map(
-  "2.11" -> "2.1.1",
   "2.12" -> "3.1.2",
   "2.13" -> "3.2.1"
```

Comment:
Probably for a separate PR, but we should probably update the Spark versions too